book/10-git-internals/sections/environment.asc

```diff
@@ -1,138 +1,138 @@
-You can override this in individual cases by starting the pathspec with `:(glob)` or `:(literal)`, as in `:(glob)*.c`.
+*`GIT_GLOB_PATHSPECS`* und *`GIT_NOGLOB_PATHSPECS`* steuern das Standardverhalten von Platzhaltern in Pfadangaben.
+Wenn `GIT_GLOB_PATHSPECS` auf 1 gesetzt ist, werden Platzhalterzeichen als Platzhalter verwendet (dies ist die Standardeinstellung). Wenn `GIT_NOGLOB_PATHSPECS` auf 1 gesetzt ist, stimmen Platzhalterzeichen nur mit sich selbst überein. Dies bedeutet, dass `*.c` nur mit einer Datei namens „* .c“ übereinstimmt und nicht mit einer Datei, deren Name mit `.c` endet.
+Sie können dies in Einzelfällen überschreiben, indem Sie die Pfadangabe mit `:(glob)` oder `:(literal)` beginnen, wie in `:(glob)*.c`.
 
-*`GIT_LITERAL_PATHSPECS`* disables both of the above behaviors; no wildcard characters will work, and the override prefixes are disabled as well.
+*`GIT_LITERAL_PATHSPECS`* deaktiviert beide oben genannten Verhaltensweisen. Es können keine Platzhalterzeichen verwendet werden, und die Präfixe zum Überschreiben sind ebenfalls deaktiviert.
 
-*`GIT_ICASE_PATHSPECS`* sets all pathspecs to work in a case-insensitive manner.
+*`GIT_ICASE_PATHSPECS`* setzt alle Pfadangaben so, dass zwischen Groß- und Kleinschreibung nicht unterschieden wird.
 
 
-==== Committing
+==== Committen
 
-The final creation of a Git commit object is usually done by `git-commit-tree`, which uses these environment variables as its primary source of information, falling back to configuration values only if these aren't present.
+Die endgültige Erstellung eines Git-Commit-Objekts erfolgt normalerweise über `git-commit-tree`, das diese Umgebungsvariablen als primäre Informationsquelle verwendet und nur dann auf Konfigurationswerte zurückgreift, wenn diese nicht vorhanden sind.
 
-*`GIT_AUTHOR_NAME`* is the human-readable name in the ``author'' field.
+* `GIT_AUTHOR_NAME` * ist der lesbare Name im Feld „author“.
 
-*`GIT_AUTHOR_EMAIL`* is the email for the ``author'' field.
+* `GIT_AUTHOR_EMAIL` * ist die E-Mail-Adresse für das Feld „author“.
 
-*`GIT_AUTHOR_DATE`* is the timestamp used for the ``author'' field.
+* `GIT_AUTHOR_DATE` * ist der Zeitstempel für das Feld „author“.
 
-*`GIT_COMMITTER_NAME`* sets the human name for the ``committer'' field.
+* `GIT_COMMITTER_NAME` * legt den echten Namen für das Feld „Committer“ fest.
 
-*`GIT_COMMITTER_EMAIL`* is the email address for the ``committer'' field.
+* `GIT_COMMITTER_EMAIL` * ist die E-Mail-Adresse für das Feld „Committer“.
 
-*`GIT_COMMITTER_DATE`* is used for the timestamp in the ``committer'' field.
+* `GIT_COMMITTER_DATE` * wird für den Zeitstempel im Feld „Committer“ verwendet.
 
-*`EMAIL`* is the fallback email address in case the `user.email` configuration value isn't set.
-If _this_ isn't set, Git falls back to the system user and host names.
+* `EMAIL` * ist die Ersatz-E-Mail-Adresse für den Fall, dass der Konfigurationswert `user.email` nicht festgelegt ist.
```

Comment:
Ich sehe im Diff in dieser Auflistung zusätzliche Leerzeichen im Vergleich zur englischen Version. Können wir das ignorieren?